boards/arm/mimxrt1024_evk/doc/index.rst

```diff
@@ -0,0 +1,212 @@
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+``boards/arm/mimxrt1024_evk/mimxrt1024_evk_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and I/Os
+====================
+
+The MIMXRT1024 SoC has five pairs of pinmux/gpio controllers.
+
++---------------+-----------------+---------------------------+
+| Name          | Function        | Usage                     |
++===============+=================+===========================+
+| GPIO_AD_B1_08 | GPIO            | LED                       |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_06 | LPUART1_TX      | UART Console              |
++---------------+-----------------+---------------------------+
+| GPIO_AD_B0_07 | LPUART1_RX      | UART Console              |
++---------------+-----------------+---------------------------+
+| WAKEUP        | GPIO            | SW4                       |
++---------------+-----------------+---------------------------+
+
+System Clock
+============
+
+The MIMXRT1024 SoC is configured to use the 24 MHz external oscillator on the
+board with the on-chip PLL to generate a 500 MHz core clock.
+
+Serial Port
+===========
+
+The MIMXRT1024 SoC has eight UARTs. One is configured for the console and the
+remaining are not used.
+
+Programming and Debugging
+*************************
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Configuring a Debug Probe
+=========================
+
+A debug probe is used for both flashing and debugging the board. This board is
+configured by default to use the :ref:`opensda-daplink-onboard-debug-probe`,
+however the :ref:`pyocd-debug-host-tools` do not yet support programming the
+external flashes on this board so you must reconfigure the board for one of the
+following debug probes instead.
+
+:ref:`jlink-external-debug-probe`
+-------------------------------------------
+
+Install the :ref:`jlink-debug-host-tools` and make sure they are in your search
+path.
+
+Attach a J-Link 10-pin connector to J55. Check that jumpers J47 and J48 are
+**off** (they are on by default when boards ship from the factory) to ensure
+SWD signals are disconnected from the OpenSDA microcontroller.
+
+Configuring a Console
+=====================
```

Comment:
The length of such makrdown must be overall same as the length of the text one line above.

Reply:
I think this comment is intended for line 133, not line 143?